modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -7,32 +7,68 @@
 = Using Data Mover for CSI snapshots
 
 :FeatureName: Data Mover for CSI snapshots
-include::snippets/technology-preview.adoc[]
 
-The OADP 1.1.2 Data Mover enables customers to back up container storage interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs. The OADP 1.1.2 Data Mover solution uses the Restic option of VolSync.
+The OADP Data Mover enables customers to back up Container Storage Interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications, using CSI volume snapshots pulled from the object store if a failure, accidental deletion, or corruption of the cluster occurs.
+
+The Data Mover solution uses the Restic option of VolSync.
 
-[NOTE]
-=====
 Data Mover supports backup and restore of CSI volume snapshots only.
 
-Currently, Data Mover does not support Google Cloud Storage (GCS) buckets.
-=====
+In OADP 1.2.x Data Mover `VolumeSnapshotBackups` (VSBs) and `VolumeSnapshotRestores` (VSRs) are queued using the VolumeSnapshotMover (VSM). The VSM's performance is improved by specifying a concurrent number of VSBs and VSRs simultaneously `InProgress`. After all async plugin operations are completed, the backup is marked as completed.
+
+
+[NOTE]
+====
+The OADP 1.2.x Data Mover is a fully supported Red Hat feature.
+
+The OADP 1.1.x Data Mover is a Technology Preview feature.
+====
+:FeatureName: The OADP 1.1.x Data Mover
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[NOTE]
+====
+Red Hat recommends that customers who use OADP 1.2.x Data Mover to back up and restore ODF CephFS volumes, upgrade or install OCP 4.12 for improved performance. OADP Data Mover can leverage CephFS shallow volumes in OCP 4.12+, which based on our testing improves the performance of backup times.
+
+* https://https://issues.redhat.com/browse/RHSTOR-4287[CephFS ROX details]
+* https://https://github.com/ceph/ceph-csi/blob/devel/docs/cephfs-snapshot-backed-volumes.md[Provisioning and mounting CephFS snapshot-backed volumes]
```

Comment:
You have double **https://https://** in both links.

Reply:
I'm not sure we should be linking to a GitHub article. Has it been vetted by OCP QE?

Reply:
> CephFS ROX details

You might want to change the link text to indicate what I might learn from the linked BZ.

Reply:
@mburke5678 @anarnold97 @akarol 
I have commented out the GitHub link until we get OCP QE verification for the next release.